common/cultures/cwp_modern_celts_cultures.txt

```diff
@@ -597,7 +597,7 @@ scottish = {
 		Annot
 		Alice
 		Amelia
-		Aiofe
+		Aoife
```

Comment:
I have a coworker with this name and the Internet seems to say that `Aiofe` is just a misspelling of `Aoife`.

Reply:
Hi! I am said coworker and Aoife should be in the Irish Catholic category, rather than Scottish Protestant. 
Also fun fact - the `O'` before Irish last names means 'son of' and `Ni'` means 'daughter of' and this system was traditionally used for all last names.
So a man today might go by Patrick Collins, but historically he would be known as Patrick O'Coileain and his sister might be Charlotte Ni'Coileain.

Reply:
Its spelled like that in vanilla